specification.md

```diff
@@ -1631,6 +1631,15 @@ There are two types of conditions for switch states:
 
 These are exclusive, meaning that a switch state can define one or the other condition type, but not both.
 
+The order on which conditions are defined must be respected by implementations unless priorities are speficied. 
+If priorities are specified, the most prioritary among satisfied conditions must have its transition executed.
+
+For example, let's say there are two conditions, A and B, defined in that order. If both A and B are satisfied
+then the A's transition must be executed. This behavior is akin to swtich statements in many programming languages.
```

Comment:
I would not put anything about "programming languages" here. However I would add that In the cases "priority" is used, and not all conditions define a priority OR certain conditions in this case have the SAME priority defined in the markup, the runtime engines are expected to notify the user about this (as this is definitely a logical workflow error)